examples/data-transfer-pg/main.py

```diff
@@ -21,18 +26,20 @@ def main():
 
         with source_db_cur, dest_db_cur:
 
-            print(f'Create fruits table and populate data in source database.')
+            print(f'Create users table and populate data in source database.')
+
             source_db_cur.execute(extract_sql('sql/source_db_setup.sql'))
+            data = get_data(RECORD_COUNT, source_db_cur)
```

Comment:
You don't need to return here. Just do.
```py
get_data(RECORD_COUNT, source_db_cur)
```

Also, rename the function to `populate_data` because you're populating data here not getting it.